lnd_test.go

```diff
@@ -291,6 +315,35 @@ func testBasicChannelFunding(net *networkHarness, t *harnessTest) {
 	closeChannelAndAssert(ctxt, t, net, net.Alice, chanPoint, false)
 }
 
+// testDisconnectingTargetPeer performs a test which
+// disconnects Alice-peer from Bob-peer and then re-connects them again
+func testDisconnectingTargetPeer(net *networkHarness, t *harnessTest) {
+
+	ctxb := context.Background()
+	timeout := time.Duration(time.Second * 10)
+	ctxt, _ := context.WithTimeout(ctxb, timeout)
+
+	// check existing connection.
```

Comment:
To be consistent with the rest of the codebase, comments should start with uppercase, same should be applied to the rest of PR.